squidb/src/com/yahoo/squidb/sql/Function.java

```diff
@@ -129,10 +129,10 @@ protected String separator() {
     }
 
     /**
-     * Create a Function that counts all rows
+     * Create a Function that counts all rows (i.e. count(*))
      */
     public static Function<Integer> count() {
-        return new ArgumentFunction<Integer>("COUNT", 1);
+        return new RawFunction<Integer>("COUNT(*)");
```

Comment:
They should be equivalent. Couldn't find anything specifically mentioning SQLite, but http://stackoverflow.com/questions/1221559/count-vs-count1. IMHO count(*) is easier to understand as "cardinality", where as sometimes when I see count(1) I say "wait, what are the rules again?"